tests/unit/test_configuration.py

```diff
@@ -21,10 +21,29 @@ def user_configuration_yaml(tmp_path):
     return user_config_path
 
 
-def test_get_configuration(user_configuration_yaml):
-    config = get_configuration()
-    assert config
-    assert isinstance(config, ConfigTree)
+def test_get_configuration(mocker):
+    """Tests that the default configuration can be retrieved."""
+    mock = mocker.patch("pseudopeople.utilities.ConfigTree")
+    _ = get_configuration()
+    mock.assert_called_once_with(
+        data=Path(pseudopeople.__file__).resolve().parent / "default_configuration.yaml",
+        layers=["base", "user"],
+    )
 
-    overridden_config = get_configuration(user_configuration_yaml)
-    assert config != overridden_config
+
+def test_get_configuration_with_user_override(user_configuration_yaml, mocker):
+    """Tests that the default configuration get updated when a user configuration is supplied."""
+    mock = mocker.patch("pseudopeople.utilities.ConfigTree")
+    _ = get_configuration(user_configuration_yaml)
+    mock.assert_called_once_with(
```

Comment:
You've copied the entirety of `test_get_configuration` into this test. Is this necessary to make the rest of this work?

Reply:
No, I didn't. The call on 37 is different. The assert on 38 is the same as the previous test but it isn't unnecessary, since it's on a different call.